src/js/messaging/components/compose/MessageSubject.jsx

```diff
@@ -9,12 +9,13 @@ class MessageSubject extends React.Component {
     return (
       <div className="messaging-subject">
         <ErrorableTextInput
+            additionalClass="messaging-subject-input"
             errorMessage={this.props.errorMessage}
-            label="Subject Line:"
+            label="Subject"
```

Comment:
"Subject Line" is in Compose.jsx as the `legend` for the `fieldset` that wraps this and the category field. This is the label for just the subject field.